untitled/Study/Decorator_Property.py

```diff
@@ -0,0 +1,47 @@
+class Triangle:
+    def __init__(self, a, b, c):
+        self.__a = a
+        self.__b = b
+        self.__c = c
+
+    @property
+    def property_perimeter(self):
```

Comment:
As the property_perimeter is already a `property`, let's rename it to `perimeter`
```suggestion
    def perimeter(self):
```